OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitVideoFragment.java

```diff
@@ -644,10 +646,9 @@ private void updateUI(int orientation) {
             } else {
                 messageContainer.setVisibility(View.GONE);
                 transcriptListView.setVisibility(View.VISIBLE);
-
                 // Calculating the offset required for centralizing the current transcript item
                 // p.s. Without this listener the getHeight function returns 0
-                transcriptListView.getViewTreeObserver().addOnGlobalLayoutListener(new ViewTreeObserver.OnGlobalLayoutListener() {
+                transcriptListLayoutListener = new ViewTreeObserver.OnGlobalLayoutListener() {
                     public void onGlobalLayout() {
                         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
```

Comment:
We are no longer supporting JellyBean users, so this check and its related logic can be removed.